src/components/BaseComponents/FormGroup/HtmlToText.tsx

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
Can you rename this file to be the name of the Component.

Also, I think it needs to be considered whether this should be part of BaseComponents, or SDK components. We have to try to keep in mind that we would like to make these base components available to wider audience at some point, so anything that is related to handling situations to do with pega should be separated out.